make_version_file.py

```diff
@@ -39,6 +39,8 @@
 	text = f.read()
 	text = text.replace("${PACKAGE_WCREF}", f"{rev_number} (Git)")
 	text = text.replace("${PACKAGE_WCDATE}", commit_date)
+	text.append('#define FCRepositoryHash   "%s"\n' % (commit_hash))
+        text.append('#define FCRepositoryBranch "%s"\n' % (branch_name))
```

Comment:
I guess the extra tab is wrong here

Reply:
possibly, I just followed the indentation above, if these lines don't need indentation, aren't some of the lines above also in the same situation?